downstream/titles/release-notes/topics/aap-24.adoc

```diff
@@ -0,0 +1,111 @@
+
+* Updated django for potential denial-of-service vulnerability in file uploads (CVE-2023-24580).
+
+* Updated sqlparse for ReDOS vulnerability (CVE-2023-30608).
+
+* Updated django for potential denial-of-service via Accept-Language headers (CVE-2023-23969).
+
+
+== Deprecated and removed features
+
+Some features available in previous releases have been deprecated or removed. Deprecated functionality is still included in Ansible Automation Platform and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments. 
+
+The following is a list of major functionality deprecated and removed within Ansible Automation Platform 2.4:
+
+* On-premise component Automation Services Catalog is now removed from Ansible Automation Platform 2.4 onwards.
+
+* With the Ansible Automation Platform 2.4 release, the Execution Environment container image for Ansible 2.9 (*ee-29-rhel-8*) is no longer loaded into the Automation Controller configuration by default.
+
+* Although you can still synchronize content, the use of synclists is deprecated and will be removed in a later release. Instead, private automation hub administrators can upload manually-created requirements files from the `rh-certified` remote.
+
+* You can now configure the Controller Access Token for each resource with the `connection_secret` parameter, rather than the old `tower_auth_secret` parameter.  This change is backwards compatible, but the `tower_auth_secret` parameter is now deprecated and will be removed in a future release.
+
+* Smart inventories have been deprecated in favor of constructed inventories and will be removed in a future release.
+
+== Bug fixes
+
+The following bugs were fixed in this release of Ansible Automation Platform:
+
+* Installer now ensures that collection auto signing cannot be enabled without enabling the collection signing service.
+
+*  Fixed an issue with restoring backups when the installed automation controller version is different from the backup version.
+
+* Fixed an issue with not adding user defined galaxy-importer settings to `galaxy-importer.cfg` file.
+
+* Added missing `X-Forwarded-For` header information to nginx logs.
+
+* Removed unnecessary receptor peer name validation when IP address is used as the name.
+
+* Updated the `outdated base_packages.txt` file that is included in the bundle installer.
+
+* Fixed an issue where upgrading the Ansible Automation Platform did not update the nginx package by default.
+
+* Fixed an issue where an *awx* user was created without creating an *awx* group on execution nodes.
+
+* Fixed the assignment of package version variable to work with flat file inventories. 
+
+* Added a FQDN check for the automation hub hostname required to run the Skopeo commands.
+
+*  Fixed an issue such that the front end URL for Red Hat Single Sign On (SSO) is now properly configured after you specify the `sso_redirect_host` variable.
```

Comment:
Extra space.

Reply:
Corrected the spacing here as well, thank you.